client/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Posts</title>
+    <link rel="stylesheet" href="./static/css/index.css">
+    <script defer src=".static/js/script.js"></script>
```

Comment:
Think I was getting an error due to the script src - does this need to be changed to `src="./static/js/script.js"` ?